pkg/runtime/unstructured/converter.go

```diff
@@ -0,0 +1,472 @@
+/*
```

Comment:
Yeah, should go to staging/src/k8s.io/apimachinery/pkg/conversion/unstructured (since this is about conversion).  That is symlinked into vendor, so your type will be available at `k8s.io/apimachinery/pkg/conversion/unstructured`.